src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Tests/Fixtures/DataSetVersionGeneratorExtensions.cs

```diff
@@ -408,81 +418,110 @@ public static InstanceSetters<DataSetVersion> SetStatusDraft(this InstanceSetter
             }
         );
 
-    public static InstanceSetters<DataSetVersion> SetFilterChanges(
+    public static InstanceSetters<DataSetVersion> SetFilterMetaChanges(
         this InstanceSetters<DataSetVersion> instanceSetter,
-        IEnumerable<ChangeSetFilters> filterChanges)
+        IEnumerable<FilterMetaChange> filterChanges)
         => instanceSetter.Set(
-            dsv => dsv.FilterChanges,
+            dsv => dsv.FilterMetaChanges,
             (_, dsv) => filterChanges.Select(
-                    changeSet =>
+                    change =>
                     {
-                        changeSet.DataSetVersionId = dsv.Id;
-                        changeSet.DataSetVersion = dsv;
-                        return changeSet;
+                        change.DataSetVersion = dsv;
+                        change.DataSetVersionId = dsv.Id;
+                        return change;
                     }
                 )
                 .ToList()
         );
 
-    public static InstanceSetters<DataSetVersion> SetFilterOptionChanges(
+    public static InstanceSetters<DataSetVersion> SetFilterOptionMetaChanges(
         this InstanceSetters<DataSetVersion> instanceSetter,
-        IEnumerable<ChangeSetFilterOptions> filterOptionChanges)
+        IEnumerable<FilterOptionMetaChange> filterOptionChanges)
         => instanceSetter.Set(
-            dsv => dsv.FilterOptionChanges,
+            dsv => dsv.FilterOptionMetaChanges,
             (_, dsv) => filterOptionChanges.Select(
-                    changeSet =>
+                    change =>
                     {
-                        changeSet.DataSetVersionId = dsv.Id;
-                        changeSet.DataSetVersion = dsv;
-                        return changeSet;
+                        change.DataSetVersion = dsv;
+                        change.DataSetVersionId = dsv.Id;
+                        return change;
                     }
                 )
                 .ToList()
         );
+    
+    public static InstanceSetters<DataSetVersion> SetGeographicLevelMetaChange(
+        this InstanceSetters<DataSetVersion> instanceSetter,
+        GeographicLevelMetaChange geographicLevelChange)
+        => instanceSetter.Set(
+            dsv => dsv.GeographicLevelMetaChange,
+            (_, dsv) =>
+            {
+                geographicLevelChange.DataSetVersion = dsv;
```

Comment:
Looks like elsewhere, you've opted to just call this parameter `change`, so you might wanna shorten this one too to match?